perfkitbenchmarker/benchmarks/object_storage_service_benchmark.py

```diff
@@ -752,6 +752,8 @@ def Prepare(self, vm):
                      '--rc-path=.bash_profile '
                      '--path-update=true '
                      '--bash-completion=true')
+    # Need to install crcmod for gcp. See "gsutil help crc" for details.
+    vm.RemoteCommand('sudo pip install -U crcmod')
```

Comment:
Yes we should be able to add a metadata entry to indicate if the crcmod is installed by default or by the benchmark. I filed issue 457 and assigned to myself. Will get to that soon.